### PR TITLE
merge from develop to master

### PR DESCRIPTION
**Pull request by Yzm1234**

### CODE REVIEW

**Comment by MrCreosote** [comment]
See review here: https://github.com/kbaseapps/HTMLFileSetServ/pull/34

The review should be addressed before merging to develop for future reference

File: test/src/htmlfilesetserv/test/HTMLFileSetServServerTest.java [hunk 1]
```diff
@@ -1029,9 +1009,9 @@ public void testFailReportDeletedShockNode() throws Exception {
 		testFail(path, TOKEN1.getToken(), 500, String.format(
 				"Workspace reported a handle error: The Handle Manager " +
 				"reported a problem while attempting to set Handle ACLs: " +
-				"Unable to set acl(s) on handles %s", handleID), "cookie");
+				"&#39;Unable to set acl(s) on handles %s&#39;", handleID), "cookie");
```
Comment (MrCreosote) [inline test/src/htmlfilesetserv/test/HTMLFileSetServServerTest.java:1012]: That's weird, but that's a bug in handle service, so nothing to be done here

**Review by MrCreosote — APPROVED** [review]
LGTM

**Comment by Yzm1234** [comment]
Sorry, I don't understand what I should do with LGTM?

**Comment by Yzm1234** [comment]
Ha, I think you mean LGTM analysis. Then could you please merge this PR?

**Comment by MrCreosote** [comment]
LGTM is a pretty standard way of saying the PR looks good: https://www.dictionary.com/e/acronyms/lgtm/

Can you not merge?

**Pull request merged**